From db437c6e9266f91323e3e3375541be0a26ccb7b7 Mon Sep 17 00:00:00 2001
From: Tony Lindgren <tony@atomide.com>
Date: Tue, 6 Dec 2011 05:45:37 +0100
Subject: [PATCH 0085/1092] ARM: 7192/1: OMAP: Fix build error for
 omap1_defconfig

Otherwise we get the following error:

In function 'omap_init_consistent_dma_size':
error: implicit declaration of function 'init_consistent_dma_size'

Signed-off-by: Tony Lindgren <tony@atomide.com>
Acked-by: Nicolas Pitre <nicolas.pitre@linaro.org>
Signed-off-by: Russell King <rmk+kernel@arm.linux.org.uk>
---
 arch/arm/plat-omap/common.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/plat-omap/common.c b/arch/arm/plat-omap/common.c
index 7a4578b..2ee6341 100644
--- a/arch/arm/plat-omap/common.c
+++ b/arch/arm/plat-omap/common.c
@@ -14,6 +14,7 @@
 #include <linux/kernel.h>
 #include <linux/init.h>
 #include <linux/io.h>
+#include <linux/dma-mapping.h>
 #include <linux/omapfb.h>
 
 #include <plat/common.h>
-- 
1.7.11.2

